cdp_scrapers/tests/test_scrapers.py

```diff
@@ -14,7 +14,7 @@
     "expected_minutes_item_in_first_meeting, expected_first_supporting_file",
     [
         (
-            datetime(2022, 1, 10),
+            datetime(2022, 1, 11),
```

Comment:
They added a sub committee meeting for January 10 since we created this test. We want to exclude that new event.